docs/docs-content/getting-started/additional-capabilities.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_label: "Palette Additional Capabilities"
+title: "Palette Additional Capabilities"
+description: "Learn more about Palette's Additional Capabilities."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["getting-started"]
+---
+
+Palette offers a range of additional capabilities designed to extend the functionality and versatility of Kubernetes
+workloads. These features comprise diverse deployment scenarios, ensuring organizations can manage their workloads
+across different environments. This can include data centers such as VMware vSphere and OpenStack, bare metal, or public
+cloud using your cloud accounts.
+
+Whether deploying workloads in remote locations with [Edge](./additional-capabilities.md#edge), securing sensitive
+information with [VerteX](./additional-capabilities.md#palette-vertex), maximizing management control with
+[self-hosted Palette](./additional-capabilities.md#self-hosted-palette), or integrating virtual machine workloads into
+Kubernetes environments with [Virtual Machine Orchestrator](./additional-capabilities.md#virtual-machine-orchestrator),
+Palette provides comprehensive solutions to meet different business needs. Learn more about each capability below.
```

Comment:
```suggestion
Whether deploying workloads in remote locations with [Edge](./additional-capabilities.md#edge), enforcing FIPS compliance with [VerteX](./additional-capabilities.md#palette-vertex), maximizing management control with
[self-hosted Palette](./additional-capabilities.md#self-hosted-palette), or integrating virtual machine workloads into
Kubernetes environments with [Virtual Machine Orchestrator](./additional-capabilities.md#virtual-machine-orchestrator),
Palette provides comprehensive solutions to meet different business needs. Learn more about each capability below.
```
I focused on FIPS compliance versus securing sensitive information, which could get us into the data encryption rabit hole. We can avoid that by focusing on the FIPS aspect 😄

Reply:
Oooo. I see :eyes: